archaius2-api/src/main/java/com/netflix/archaius/api/Layer.java

```diff
@@ -0,0 +1,80 @@
+
+package com.netflix.archaius.api;
+
+/**
+ * Key used to group and order {@link Config}s into layers (@see Layers).
+ * Layers are ordered by natural order such that lower order values have 
+ * precedence over a higher value.  Within a layer configurations are prioritized by
+ * insertion order (or reversed if 'reversed=true')
+ */
+public final class Layer {
+    private final String name;
+    private final int order;
+    private final boolean reversed;
```

Comment:
Why do we need to keep this concept around? Is this truly necessary for a new API?

Reply:
I suppose we don't need this and can add it later if the need should arise.